src/scripts/doAcct.sh

```diff
@@ -34,10 +54,33 @@ else
    sMon=$((eMon-1))
    sYear=$eYear
 fi
+doCut=0
 startDate=$(date -d "$sYear"/"$sMon"/"$day" +"%b %d %Y")
 endDate=$(date +"%b %d %Y")
 
-if [[ $# -gt 0 ]] ;
+# process flag args
+while [ $# -gt 0 ]; do
+    key="$1"
+    case $key in
+        -cut | cut)        doCut=1; ;;
+        -f | -from)        startDate=$(date -d "$2" +"%b %d %Y"); shift ;;
+        -t | -to)          endDate=$(date -d "$2" +"%b %d %Y"); shift ;;
+        -h | -help)        usage ;;
+        *)
+            # unknown option
+            echo "unrecognized argument: $key"
+            usage
+            ;;
+    esac
+    shift
+done
+
+if [[ "x${startDate}" = "x" ]] || [[ "x${endDate}" = "x" ]] ;
```

Comment:
nice flags handling! :-)

the semicolon is only necessary if there's no newline between `]]` and `then`